src/Behat/Behat/Transformation/Transformer/RepositoryArgumentTransformer.php

```diff
@@ -180,6 +200,35 @@ private function isApplicableTableTransformation(Transformation $transformation,
     }
 
     /**
+     * Checks if provided transformation is applicable row table transformation.
+     *
+     * @param Transformation $transformation
+     * @param mixed          $value
+     *
+     * @return Boolean
+     */
+    private function isApplicableRowTableTransformation(Transformation $transformation, $value)
+    {
+        if (!$value instanceof TableNode) {
+            return false;
+        };
+
+        try {
```

Comment:
It reads a bit strange but the logic is fine. Maybe just add a line of documentation to the second try/catch to explain it?